CHANGELOG.md

```diff
@@ -37,6 +40,10 @@ IMPROVEMENTS:
 BUG FIXES:
 * Control Plane
    * Don't incorrectly diff intention config entries when upgrading from Consul pre-1.12 to 1.12+ [[GH-1804](https://github.com/hashicorp/consul-k8s/pull/1804)]
+  * Add discover binary to control-plane image [[GH-1749](https://github.com/hashicorp/consul-k8s/pull/1749)]
+* Helm:
+  * Don't pass in a CA file to the API Gateway controller when `externalServers.useSystemRoots` is `true`. [[GH-1743](https://github.com/hashicorp/consul-k8s/pull/1743)]
```

Comment:
These changelog entries read weird to me, is this an instruction the the user?

Reply:
It's a copy and paste from the 1.1 CHANGELOG

Reply:
all good